Lib/ufo2ft/__init__.py

```diff
@@ -635,7 +639,7 @@ def compileVariableTTF(
         inplace=inplace,
         debugFeatureFile=debugFeatureFile,
         notdefGlyph=notdefGlyph,
-        _reloadFont=False,
+        postProcessorClass=None,
```

Comment:
add a comment why we pass None here. Maybe also don't bother setting arguments that would be forwarded to PostProcessor class if it were not None, since they are ignored anyway

Reply:
I believe the offending parameters are erased already: production names and subroutinization for CFFs.